scripts/lmdk/tools/header_pack.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+import json
+import pathlib
+import shutil
+
+
+# Headers for needs of lmdk are defined in
+#            lmdk/include/headers_list.json
+
+
+SOF_TOP = pathlib.Path(__file__).parents[3].resolve()
+LMDK_HEADERS = SOF_TOP / "lmdk" / "include" / "headers_manifest.json"
+
+
+def create_separate_headers():
+    f = open(LMDK_HEADERS)
+    data = json.load(f)
+    src = ''
+
+    for i in data:
+        src = ''
+        for p in i[:-1]:
+            src += p
+            src += "/"
```

Comment:
You don't need that second loop, you can just pass the entire list 

https://docs.python.org/3/library/pathlib.html#pure-paths
```
Path('foo', 'some/path', 'bar')
==> Path('foo/some/path/bar')
```

To get the dirname use `.parent` and to get the basename use `.name`. It will be so much more readable than slicing the JSON array.